pipeline/load/display_metrics.py

```diff
@@ -1,59 +1,34 @@
-import operator
+from typing import Dict, List
 
 import prefect
 from prefect import task
 
-from pipeline.types import MatchingMetrics, MatchingOutput, User
+from pipeline.types import MatchingMetrics, MatchingOutput, User, UserId
+
+
+def render_counts_per_user(
+    users: List[User], counts: Dict[UserId, int]
```

Comment:
I only pass in `users` and `counts`, since I don't need the rest of the matching output or the metrics output.